extensions/wikia/AdEngine/js/SlotTweaker.js

```diff
@@ -31,6 +31,7 @@ define('ext.wikia.adEngine.slotTweaker', [
 		var slot = document.getElementById(slotname);
 
 		if (slot) {
+			removeClass(slot, 'hidden');
 			slot.className += ' hidden';
```

Comment:
Is it better than checking if class exists and adding only if it doesn't? Is it possible that this will cause flashing?